exp/addons/internal/controllers/clusterresourcesetbinding_controller.go

```diff
@@ -91,6 +93,12 @@ func (r *ClusterResourceSetBindingReconciler) Reconcile(ctx context.Context, req
 	}
 	// If the owner cluster is in deletion process, delete its ClusterResourceSetBinding
 	if !cluster.DeletionTimestamp.IsZero() {
+		if feature.Gates.Enabled(feature.RuntimeSDK) && feature.Gates.Enabled(feature.ClusterTopology) {
+			if cluster.Spec.Topology != nil && !hooks.IsOkToDelete(cluster) {
+				// If the Cluster is not yet ready to be deleted then do no delete the ClusterResourceSetBinding.
```

Comment:
```suggestion
				// If the Cluster is not yet ready to be deleted then do not delete the ClusterResourceSetBinding.
```

Reply:
This chagne is from the `BeforeClusterDelete` hook PR. I will address it in the other PR.